### ISSUE TITLE
Fix S2699: Rule doesn't raise when the code contains any invocation

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-dotnet#2130]
### Description

Since the tests refactoring the rule is broken and no longer raises any issue on methods containing invocations.

### Repro steps

```csharp
namespace UnitTestProject1
{
    public class Class1
    {
        public int Add(int a, int b) => a + b;
    }

    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1() // No issue raised
        {
            var result = new Class1().Add(1, 2);
        }
    }
}
```
